docker/release/anolis_entrypoint.py

```diff
@@ -0,0 +1,36 @@
+banner=f"""
+Welcome to use
+   _____                     __  ________             
+  / ___/___  _____________  / /_/ ____/ /___ _      __
+  \__ \/ _ \/ ___/ ___/ _ \/ __/ /_  / / __ \ | /| / /
+ ___/ /  __/ /__/ /  /  __/ /_/ __/ / / /_/ / |/ |/ / 
+/____/\___/\___/_/   \___/\__/_/   /_/\____/|__/|__/  
+"""
+print(banner)
+import os
```

Comment:
Can you sort the imports? Please refer to [imports-formatting](https://google.github.io/styleguide/pyguide.html#313-imports-formatting)

Reply:
importing tensorflow directly will bring the warning message, using the line to change the level of logging to block information of warnings before importing tensorflow
```
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Reply:
The imports are not in alphabetical order still.
`jax` && `jaxlib` should be in the front of `secretflow`